tests/src/tools/ReadyToRun.SuperIlc/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.CommandLine.Builder;
+using System.CommandLine.Invocation;
+using System.Threading.Tasks;
+
+namespace ReadyToRun.SuperIlc
+{
+    class Program
+    {
+        static async Task<int> Main(string[] args)
+        {
+            var parser = CommandLineOptions.Build()
+                
+                .UseHelp()
+                .UseParseDirective()
+                .UseDebugDirective()
+                .UseSuggestDirective()
+                .RegisterWithDotnetSuggest()
+                .UseParseErrorReporting()
+                .UseExceptionHandler()
+                .UseTypoCorrections()
+
+                .Build();
+
+            return await parser.InvokeAsync(args);
```

Comment:
There is no synchronous way to run the stuff?

Reply:
Hmm, it does not look like it. It does not sound right that they are forcing async onto everybody...

Reply:
Currently their API to automatically parse the command line and call the command handler is done async only. The synchronous alternative is to call `Parser.Parse` and pick through the set arguments / options manually.